experimenter/experimenter/jetstream/tests/test_tasks.py

```diff
@@ -2168,29 +2168,32 @@ def test_results_data_null(self, lifecycle):
                     "experiment": f"{experiment.slug}",
                     "filename": "experimenter/jetstream/client.py",
                     "func_name": "load_data_from_gcs",
-                    "log_level": "ERROR",
+                    "log_level": "WARNING",
                     "message": f"Could not find data in analysis bucket at path statistics/statistics_{experiment.slug.replace('-', '_')}_weekly.json",  # noqa: E501
                     "metric": None,
                     "segment": None,
                     "statistic": None,
                     "timestamp": now.isoformat(timespec="milliseconds"),
                 },
-                {
-                    "analysis_basis": None,
-                    "source": None,
-                    "exception": None,
-                    "exception_type": None,
-                    "experiment": f"{experiment.slug}",
-                    "filename": "experimenter/jetstream/client.py",
-                    "func_name": "load_data_from_gcs",
-                    "log_level": "ERROR",
-                    "message": f"Could not find data in analysis bucket at path statistics/statistics_{experiment.slug.replace('-', '_')}_overall.json",  # noqa: E501
-                    "metric": None,
-                    "segment": None,
-                    "statistic": None,
-                    "timestamp": now.isoformat(timespec="milliseconds"),
-                },
             ]
+            if lifecycle == NimbusExperimentFactory.Lifecycles.ENDING_APPROVE_APPROVE:
+                experiment_errors.append(
+                    {
+                        "analysis_basis": None,
+                        "source": None,
+                        "exception": None,
+                        "exception_type": None,
+                        "experiment": f"{experiment.slug}",
```

Comment:
nit: you don't need to fstring here, or in any of ther other locations

```suggestion
                        "experiment": experiment.slug,
```

Reply:
Ah yea good point. Feels like a copy paste remnant.. Thanks!